tests/framework/types.go

```diff
@@ -19,6 +24,9 @@ type OSMDescribeInfo struct {
 	// Bucket indicates in which test Bucket the test will run in for CI. Each
 	// Bucket is run in parallel while tests in the same Bucket run sequentially.
 	Bucket int
+
+	// OSBitMask indicates which OS the test can run on.
+	OSBitMask uint8
```

Comment:
`OS string` doesn't work?

Reply:
string is a bit weird because it can have three values, Windows linux and cross-platform, so I went with a representation that could capture all three in one variable

Reply:
string can also work but it seems that it might be more error prone. if you think its better as a string, I dont have a strong opinion

Reply:
If I were writing C code I would have done the same, but given this is in the test code and will be integrated into the CI, I think it would be better to make this human-readable and reference it in the workflows file as `linux, windows, crossPlatform` when invoking tests based on the OS.

Reply:
If you were interested, you can also create an enum in Go and give it a String() method for readability, but this is fine for now.